File: BuilderDesignPattern/Desktop_Example/Products/Desktop.java
package BuilderDesignPattern.Desktop_Example.Products;


public class Desktop {

  private String monitor;
  private String keyboard;
  private String mouse;
  private String speakers;
  private String ram;
  private String processor;
  private String motherboard;

  public void setMonitor(String monitorName) {
    this.monitor = monitorName;
  }

  public void setKeyboard(String keyboardName) {
    this.keyboard = keyboardName;
  }

  public void setMouse(String mouseName) {
    this.mouse = mouseName;
  }

  public void setSpeakers(String speakersName) {
    this.speakers = speakersName;
  }

  public void setRam(String ramName) {
    this.ram = ramName;
  }

  public void setProcessor(String processorName) {
    this.processor = processorName;
  }

  public void setMotherboard(String motherboardName) {
    this.motherboard = motherboardName;
  }

  @Override
  public String toString() {
    return "Desktop: {\n" +
             "	monitor = " + monitor + "\n" +
             "	keyboard = " + keyboard + "\n" +
             "	mouse = " + mouse + "\n" +
             "	speakers = " + speakers + "\n" +
             "	ram = " + ram + "\n" +
             "	processor = " + processor + "\n" +
             "	motherboard = " + motherboard + "\n" +
             "}";
  }
}

File: BuilderDesignPattern/Destop_Example_Python/Products/Desktop.py
class Desktop:

	_monitor: str
	_keyboard: str
	_mouse: str
	_speaker: str
	_ram: str
	_processor: str
	_motherboard: str

	def setMonitor(self, monitor):
		self._monitor = monitor

	def setKeyboard(self, keyboard):
		self._keyboard = keyboard
	
	def setMouse(self, mouse):
		self._mouse = mouse
	
	def setSpeakers(self, speaker):
		self._speaker = speaker
	
	def setRam(self, ram):
		self._ram = ram
	
	def setProcessor(self, processor):
		self._processor = processor
	
	def setMotherboard(self, motherboard):
		self._motherboard = motherboard
	
	def __repr__(self):
		return f"""Desktop: {{
					monitor = {self._monitor}
					keyboard = {self._keyboard}
	 				mouse = {self._mouse}
	 				speakers = {self._speaker}
	 				ram = {self._ram}
	 				processor = {self._processor}
 					motherboard = {self._motherboard}
            		}}"""
	
	def __str__(self):
		return f"""Desktop: {{
					monitor = {self._monitor}
					keyboard = {self._keyboard}
			 		mouse = {self._mouse}
			 		speakers = {self._speaker}
			 		ram = {self._ram}
			 		processor = {self._processor}
		 			motherboard = {self._motherboard}
		            }}"""